networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/UpstreamAttestationPipe.java

```diff
@@ -0,0 +1,7 @@
+package tech.pegasys.teku.networking.eth2.gossip.topics;
+
+import tech.pegasys.teku.datastructures.attestation.ValidateableAttestation;
+
+public interface UpstreamAttestationPipe {
```

Comment:
Need to think about how we name this.  It's not really a pipe, but it's also not really a handler. Maybe just `GossipedAttestationConsumer`?  Not sure...

Reply:
I like GossipedAttestationConsumer. I made the change. Although, I have to say, `UpstreamAttestationPipe` really made me feel like an old engineer dealing with serious plumbing issues :D